Ollivanders/src/net/pottercraft/Ollivanders2/Potion/IngredientType.java

```diff
@@ -7,35 +7,51 @@
 
 public enum IngredientType
 {
+   ACONITE (Material.SPIDER_EYE, (short)0, "Aconite"),
    ARMADILLO_BILE (Material.POTION, (short)9, "Armadillo Bile"),
+   ASPHODEL (Material.DEAD_BUSH, (short)0, "Asphodel"),
```

Comment:
We already have POWDERED_ASPHODEL. If the potion wants the whole root, we probably want to name this ASPHODEL_ROOT/"Root of Asphodel" since Asphodel as a plant is a flower (so wouldn't look like a dead bush).

Reply:
Removed.